File: Nodo.py
class Nodo:
    def __init__(self, nombre: str, descripcion: str, nivel: int, acertijo: str, respuesta : str, izquierda=None, derecha=None):
        self.nombre = nombre
        self.descripcion = descripcion
        self.nivel = nivel
        self.acertijo = acertijo
        self.respuesta = respuesta
        self.izquierda = izquierda
        self.derecha = derecha

File: LAB1/src/lab1/Nodo.java
package lab1;

public class Nodo {

    String nombre;
    String descripcion;
    String acertijo;
    String respuesta;
    int dificultad;
    Nodo izquierda;
    Nodo derecha;

    public Nodo(String nombre, String descripcion, String acertijo, String respuesta, int dificultad) {
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.acertijo = acertijo;
        this.respuesta = respuesta;
        this.dificultad = dificultad;
        this.izquierda = null;
        this.derecha = null;
    }
}
